birthday/README.md

```diff
@@ -1,92 +1,90 @@
-Birthday CC V2.0
-================
+# Birthdays
+This custom command adds birthday functionality to your server, wishing members all the best on their respective date.
 
-# Info
+## Features
+* Send a configurable message in the chat on birthdays
+* Kick or ban users younger than 13 years
```

Comment:
Maybe add: `if so you want` here

Reply:
More tending towards -1 on that, should become evident when reading the documentation...
The feature list was purely meant as a small selection of features, whether configurable or optional doesn't matter.